src/condor_tests/test_startd_cron.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env pytest
+
+#
+# Test startd cron and STARTD_CRON_LOG_NON_ZERO_EXIT
+
+import pytest
+import logging
+import time
+
+import htcondor
+
+from ornithology import (
+    config,
+    standup,
+    action,
+    Condor,
+    write_file,
+    JobID,
+    SetJobStatus,
+    JobStatus,
+)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+goodcronscript = """#!/usr/bin/env python3
+import sys
+print("GoodCron = 123\\n")
+sys.stderr.write("GoodCron writes to stderr\\n")
+sys.exit(0)
+"""
+
+badcronscript = """#!/usr/bin/env python3
+import sys
+print("BadCron = 123\\n")
+sys.stderr.write("BadCron writes to stderr\\n")
+sys.exit(37)
+"""
+
+@standup
+def cronscripts(test_dir):
+    write_file(test_dir / "goodcron.py", goodcronscript)
+    write_file(test_dir / "badcron.py",  badcronscript)
+
```

Comment:
It would be traditional to return the tuple naming the good script and the bad script here, just so you don't have to type "goodcron.py" and "badcron.py" again.